styles/style.css

```diff
@@ -0,0 +1,691 @@
+@import url('http://fonts.cdnfonts.com/css/cocogoose');
+@import url('https://fonts.googleapis.com/css2?family=Poppins:wght@300;400;500;600;700&display=swap');
```

Comment:
Please change this font to `Lato`, as it is a project requirement.

<img width="495" alt="Screen Shot 2022-08-04 at 12 49 19" src="https://user-images.githubusercontent.com/82426682/182917195-5eeffdc0-e8eb-4f94-90cf-3b7f07dd1987.png">